start

```diff
@@ -2,15 +2,15 @@
 
 from lib.game import *
 from lib.board import *
-from lib.human_player import HumanPlayer
+from lib.game_selection import GameSelection
 from lib.ui import *
 from lib.tictactoe_board import *
 
 ui = UI()
 board = Board(ui)
```

Comment:
I think this was missed in my previous PR feedback, but could you remove this and its import since it's no longer being used?

Reply:
done